scene/gui/tab_bar.h

```diff
@@ -38,6 +38,10 @@ class TabBar : public Control {
 	GDCLASS(TabBar, Control);
 
 public:
+	enum TabTitleStyle {
+		NORMAL,
+		ITALIC //TODO: add bold support
```

Comment:
```suggestion
		NORMAL_FONT,
		ITALIC_FONT //TODO: add bold support
```

Too simple names like `NORMAL`, `ITALIC` and `BOLD` can conflict with some system headers, so it's better to avoid it.

Reply:
🤔